ci/jenkins/docker_login.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+# Copyright 2024 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DOCKER_REGISTRY=$(head -n1 "${WORKSPACE}/ci/docker-registry")
+
+_usage="Usage: $0 [--docker-user <dockerUser>] [--docker-password <dockerPassword>]
+Run Docker login script.
+        --docker-user             Username for Docker account.
+        --docker-password         Password for Docker account.
+        --registry                The image registry to use."
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    --docker-user)
+    DOCKER_USERNAME="$2"
+    shift 2
+    ;;
+    --docker-password)
+    DOCKER_PASSWORD="$2"
+    shift 2
+    ;;
+    --registry)
+    DOCKER_REGISTRY="$2"
+    shift 2
+    ;;
+    -h|--help)
+    print_usage
+    exit 0
+    ;;
+    *)    # unknown option
+    echoerr "Unknown option $1"
+    exit 1
+    ;;
+esac
+done
+
+function docker_login() {
+    set +ex
+    for i in `seq 5`; do
+        output=$(echo $2 | docker login --username=$1 --password-stdin 2>/dev/null)
+        # Check if standard output contains "Login Succeeded"
+        if [[ "$output" == *"Login Succeeded"* ]]; then
+            echo "Docker login successful."
+            break
+        else
+            sleep 5s
+            echo "Docker login failed. Retrying"
```

Comment:
could it log the original output if it fails? so we know what's wrong, networking issue or incorrect password.